content/learn/node-request-data.md

```diff
@@ -20,29 +18,7 @@ axios.post('https://whatever.com/todos', {
 });
 ```
 
-This is the matching server-side code:
```

Comment:
Please add an example with the `http` module. Similar to what you added on `node-terminate-program` 🤔